Makefile

```
@@ -150,10 +150,21 @@ typescript: prepare_lib
 	# Copy generated files to the frontend
 	mkdir -p frontend/src/Generated
 	cp $(TS_OUT)/segmented-cfmm/src/common.ts frontend/src/Generated/common.ts
-	cp -r $(TS_OUT)/segmented-cfmm/src/generated/ frontend/src/Generated/generated/
+	cp -a $(TS_OUT)/segmented-cfmm/src/generated frontend/src/Generated/
 
 # Quickly setup the frontend and run the it locally. Check `frontend/readme.md` for details.
-frontend:
+frontend: env = development
+frontend: testnet_link = https://granada.testnet.tezos.serokell.team/
+frontend: mainnet_link = https://mainnet.api.tez.ie/
```

These should probably be named:
```suggestion
frontend: testnet_url = https://granada.testnet.tezos.serokell.team/
frontend: mainnet_url = https://mainnet.api.tez.ie/
```
note: if you agree please update them in the README too